CI/circle_parallel.sh

```diff
@@ -43,6 +43,21 @@ elif [ "$NODE_INDEX" = "2" ]; then
 
   # run integration tests
   mvn --no-snapshot-updates --quiet verify -Psamples.misc -Dorg.slf4j.simpleLogger.defaultLogLevel=error
+elif [ "$NODE_INDEX" = "3" ]; then
+
+  echo "Running node $NODE_INDEX to test 'samples.circleci.node3' defined in pom.xml ..."
+  #wget https://www.python.org/ftp/python/3.8.9/Python-3.8.9.tgz
+  #tar -xf Python-3.8.9.tgz
+  #cd Python-3.8.9
+  #./configure --enable-optimizations
+  #sudo make altinstall
+  pyenv install --list 
+  pyenv install 3.6.3
+  pyenv global 3.6.3
+  python3 --version
+
+  mvn --no-snapshot-updates --quiet verify -Psamples.circleci.node3 -Dorg.slf4j.simpleLogger.defaultLogLevel=error
```

Comment:
Should the other circleci samples be renamed as node1, node2 etc rather than misc?
Or maybe something like node2_javasampleses

Reply:
Will do it in another PR instead. So many things need to be cleanup in the CI 😭